### PR TITLE
interfaces/builtin: add serial devices (ttyLPX) to allowed list

### PR DESCRIPTION
**Pull request by AristoChen**
This commit is for the serial port of NXP Layerscape SoC

The prefix of the device name can be found [here](https://elixir.bootlin.com/linux/latest/source/drivers/tty/serial/fsl_lpuart.c#L245)

### CODE REVIEW

File: interfaces/builtin/serial_port.go [hunk 1]
```diff
@@ -76,7 +76,8 @@ func (iface *serialPortInterface) String() string {
 //   - ttyMSMX (Qualcomm msm7x serial devices)
 //   - ttyHSX (Qualcomm GENI based QTI serial cores)
 //   - ttyGSX (USB gadget serial devices)
-var serialDeviceNodePattern = regexp.MustCompile("^/dev/tty(mxc|USB|ACM|AMA|XRUSB|S|O|SC|MSM|HS|GS)[0-9]+$")
+//   - ttyLPX (UART serial ports)
```
Comment (ernestl) [inline interfaces/builtin/serial_port.go:79]: Should we provide a more specific description?  e.g. NXP Low Power ...

**Review by ernestl — COMMENTED** [review]
One minor comment

**Review by bboozzoo — APPROVED** [review]
LGTM

**Review by ernestl — APPROVED** [review]
Looks good, thanks

**Pull request merged**